packages/ui/react-ui-form/src/components/Form/Form.tsx

```diff
@@ -101,12 +101,13 @@ export const Form = <T extends BaseObject>({
     <div role='form' className={mx('flex flex-col w-full gap-2 py-2', classNames)}>
       {properties
         .map((property) => {
-          const { ast, name, type, format, title, description, examples, options } = property;
+          const { ast, name, type, format, title, description, examples, options, array } = property;
           const key = [...path, name];
           const label = title ?? pipe(name, capitalize);
```

Comment:
here's the logic we discussed re table headers. perhaps factor this out (to sdk schema).
it's a trivial function, but having something in common with table might be helpful as things evolve.